cranelift/codegen/src/machinst/mod.rs

```diff
@@ -277,6 +279,9 @@ pub trait MachInstEmit: MachInst {
     type State: MachInstEmitState<Self>;
     /// Constant information used in `emit` invocations.
     type Info: MachInstEmitInfo;
+    /// Unwind info generator.
+    #[cfg(feature = "unwind")]
+    type UnwindInfo: UnwindInfoGenerator<Self>;
```

Comment:
This trait is public so conditionally adding this without default would break outside implementors of this trait that expect to have `unwind` disabled when it isn't due to another crate.

Reply:
I use unwind info for `.eh_frame` generation in cg_clif.

Reply:
It pulls in gimli and some users may not use unwind info. I don't know if `#[cfg(feature = "unwind")]` should be kept or removed.

Reply:
Addressed it in d807af3 by keeping this member mandatory to implement.